libs/utils/utils.js

```diff
@@ -751,7 +769,7 @@ function decorateSection(section, idx) {
     blocks: [...links, ...blocks],
     el: section,
     idx,
-    preloadLinks: blockLinks.autoBlocks,
+    preloadLinks: removeDuplicatedLinkBlocks(blockLinks.autoBlocks),
```

Comment:
Hi @mirafedas - I'm thinking whether we should not add the modals in the first place instead of removing them. Maybe in https://github.com/adobecom/milo/blob/stage/libs/blocks/modal/modal.js#L149 we can check whether a modal with the same ID has been already added and if so, skip loading it.